### DISCUSSION TITLE
Need help to access external storage

### DISCUSSION THREAD

**Discussion by Bwapt** [termux/termux-app#3222] [Q&A] [answered]
Hi !

I am currently trying to change rights of a folder in root in order to access my USB drive but it doesn't seems working as expected. Let me explain my thoughts : 

Because I can't run files on internal storage, I wanted to use the Termux root to run my programs, it works well, but, for an unknown reason, I can't launch files like PDF with `termux-open` in the same folder (in Termux root). So I was searching for an alternative, and the only way that seems to work was to use a SD Card or a USB stick. So I bought a USB stick, but its path is currently `/mnt/media_rw/6XXX-XXXX`, so no `/storage/external-1` (`termux-setup-storage` worked only the first time). So I am currently trying to create a Symlink of its path in the current folder, but each time I tried to open it I got a `Permission Denied` error. So I tried to change the rights of its folder by using `chmod 777 /mnt/media_rw/6XXX-XXXX`, but it is not even working !

You know now why I am basically desperate, so if anyone has an idea of the way to solve this, or has an alternative for me to be able to work on my tablet, it would be greatly appreciated...

I precise that I have a Samsung Galaxy Tab S7 FE.

**Comment by sylirre** [comment, marked as answer]
`chmod` doesn't work with FAT-compatible file systems.

A secondary partition with EXT4 on your sd card, usb drive or whatever you want to use would be an option since you already have a root.

**Comment by wei-z-git** [comment]
Hello @Bwapt , did you succeed? 
I format my usb device to EXT4 as sylirre said and it works but only works in `root` user(picture1), 
when back to normal user `u0_a311`, I still can`t access to the USB device(picture 2)
![image](https://github.com/termux/termux-app/assets/32572815/2fa59a25-c81e-422b-87db-0f17672ee066)

![image](https://github.com/termux/termux-app/assets/32572815/6fb62015-229b-4161-bfa9-cd1de98f8901)

**Reply by Bwapt** [reply]
Hi ! 
Sorry it will surely disappoint you, but I've never tried to resolve it.
Hope you'll find a way for your problem.
Bwapt
